src/main/java/org/broadinstitute/hellbender/cmdline/GATKPlugin/GATKReadFilterPluginDescriptor.java

```diff
@@ -187,6 +187,23 @@ public boolean isDependentArgumentAllowed(final Class<?> dependentClass) {
         return filters;
     }
 
+    /**
+     * Get the list of default plugins used for this instance of this descriptor. Used for help/doc generation.
+     * @return List of T
+     */
+    @Override
+    public List<Object> getDefaultInstances() { return new ArrayList<>(toolDefaultReadFilters.values()); }
+
+    /**
+     * Pass back the list of ReadFilter instances that were actually seen on the
+     * command line in the same order they were specified. This list does not
+     * include the tool defaults.
```

Comment:
Method comment makes no sense -- this method returns a single `Class` object, not a `List`